demos/common/python/openvino/model_zoo/model_api/models/detection_model.py

```diff
@@ -18,9 +18,9 @@
 
 
 class DetectionModel(ImageModel):
-    '''A abstract detection model class
+    '''An abstract detection model class
 
-    This class supports the detection models. The Detection Model should have single image input.
+    This class supports detection models. The Detection Model must have single image input.
```

Comment:
questionable statement, how number of inputs dictates model task class?

Reply:
It is vice versa - the model task dictates the number of inputs.